.github/workflows/automation-trigger-test.yml

```diff
@@ -1,18 +1,100 @@
-        github-token: ${{ secrets.BRANCHLET_ACCESS_TOKEN_PUBLIC  }}
-        script: |
-          await github.rest.actions.createWorkflowDispatch({
-            owner: 'BranchMetrics',
-            repo: 'qentelli-saas-sdk-testing-automation',
-            workflow_id: 'ios-maven.yml',
-            ref: 'master'
-          })
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Install the Apple certificate and provisioning profile
+        env:
+          BUILD_CERTIFICATE_BASE64: ${{ secrets.BUILD_CERTIFICATE_BASE64 }}
+          P12_PASSWORD: ${{ secrets.P12_PASSWORD }}
+          BUILD_PROVISION_PROFILE_BASE64: ${{ secrets.BUILD_PROVISION_PROFILE_BASE64 }}
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+        run: |
+          # create variables
+          CERTIFICATE_PATH=$RUNNER_TEMP/build_certificate.p12
+          KEYCHAIN_PATH=$RUNNER_TEMP/app-signing.keychain-db
+          PP_PATH=$RUNNER_TEMP/build_pp.mobileprovision
+
+          # import certificate 
+          echo -n "$BUILD_CERTIFICATE_BASE64" | base64 --decode -o $CERTIFICATE_PATH
+          echo -n "$BUILD_PROVISION_PROFILE_BASE64" | base64 --decode -o $PP_PATH
+
+          # create temporary keychain
+          security create-keychain -p "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+          security set-keychain-settings -lut 21600 $KEYCHAIN_PATH
+          security unlock-keychain -p "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+
+          # import certificate to keychain
+          security import $CERTIFICATE_PATH -P "$P12_PASSWORD" -A -t cert -f pkcs12 -k $KEYCHAIN_PATH
+          security list-keychain -d user -s $KEYCHAIN_PATH
+
+          # apply provisioning profile
+          mkdir -p ~/Library/MobileDevice/Provisioning\ Profiles
+          cp $PP_PATH ~/Library/MobileDevice/Provisioning\ Profiles
+      
+      - name: Create and export DeepLinkDemo IPA file
+        run: |
+          ./scripts/prep_static_xcframework.sh
+          cd ./DeepLinkDemo
+          mkdir Framework
+          unzip ../Branch_static.zip -d ./Framework/
+          pod install
+          xcodebuild -workspace DeepLinkDemo.xcworkspace -scheme DeepLinkDemo  -sdk iphoneos archive -archivePath ./IPA/DeepLinkDemo.xcarchive
+          xcodebuild -exportArchive -archivePath ./IPA/DeepLinkDemo.xcarchive -exportOptionsPlist IPA/Info.plist -exportPath IPA/
+      - name: Upload IPA to Browserstack
+        run: |
+           response=$(curl -X POST https://api-cloud.browserstack.com/app-automate/upload -u "${{ secrets.BROWSER_STACK_USER }}:${{ secrets.BROWSER_STACK_KEY }}"  -F "file=@DeepLinkDemo/IPA/DeepLinkDemo.ipa")
+           parsed=$(echo $response | jq ".app_url")
+           echo "::add-mask::$parsed"
+           echo "APP_URL=$parsed" >> "$GITHUB_ENV"
+      - name: Cheout Automation Code
+        uses: actions/checkout@v3
+        with:
+          repository: BranchMetrics/qentelli-saas-sdk-testing-automation
+          ref: gdeluna-branch/RemoveLoggingAppUrl
```

Comment:
If this branch looks good let's merge it to master, then we can merge these PRs.

Reply:
Branch `RemoveLoggingAppUrl` has already been merged so we can use the master which is the default branch.
Also, can we change the env variable APP_URL to be more specific like `BROWSER_STACK_APP_URL`